src/index.js

```diff
@@ -59,7 +60,10 @@ export default class KeyboardAccessory extends Component {
 
   render() {
     let { bottom } = this.state;
-    let { children, backgroundColor } = this.props;
+    let { children, backgroundColor, verticalOffset } = this.props;
+    
+    //adds verticalOffset prop to bottom
```

Comment:
Could you remove the comment? coz the name of the variable already explained its usage.